bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.csproj

```diff
@@ -89,7 +89,7 @@
   </ItemGroup>
 
   <PropertyGroup Condition="'$(Version)'==''">
-    <Version>0.0.0-local</Version>
+    <Version>1.1.0</Version>
```

Comment:
this is not really needed, but I think we can use this field until we will automate publishing steps